crates/bevy_audio/src/audio.rs

```diff
@@ -345,40 +133,39 @@ impl PlaybackSettings {
     }
 }
 
-#[derive(Clone)]
-pub(crate) struct SpatialSettings {
+/// Settings for playing spatial audio.
+///
+/// These settings are applied when the sound starts playing. If they are changed after
+/// the sound has started playing, they will have no effect. Bevy does not yet support
+/// moving positional sound sources.
```

Comment:
Actually, I not sure about what exactly is supported. The "positional" sound is done very weirdly (thanks rodio). You specify coordinates of the "left ear" and "right ear", and it simply varies the volume of each stereo channel according to the distance to those ear positions. It seems wildly unpractical to actually use. Our positional audio examples work, but if you look, they dont (cannot??) move the "ear positions". The transform of the orbiting object causes the positional audio to change volume on the two stereo channels (so we do support "moving objects"), but every sound playback has to hardcode the "ear positions".

I can remove this part of the doc comment, because it is more confusing than it is helpful, and not exactly true.